CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -0,0 +1,74 @@
+# Changelog 1.9
+
+## 1.9.0 - 2022-10-21
+
+### Added
+
+- Add the following capabilities to the Multi-cluster feature:
+  * Add support for Pod to Pod connectivity across clusters. ([#4219](https://github.com/antrea-io/antrea/pull/4219), [@hjiajing])
+  * Add active-passive mode high availability support for Gateway Nodes. ([#4069](https://github.com/antrea-io/antrea/pull/4069), [@luolanzone])
+  * Allow Pod IPs as Endpoints of Multi-cluster Service; option `endpointIPType` is added to Multi-cluster Controller ConfigMap to specify the Service Endpoints type. ([#4198](https://github.com/antrea-io/antrea/pull/4198), [@luolanzone])
```

Comment:
to the Multi-cluster Controller ConfigMap

Reply:
updated